iceoryx_hoofs/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -77,12 +79,14 @@ UnixDomainSocket::UnixDomainSocket(const IpcChannelName_t& name,
 {
 }
 
-// NOLINTNEXTLINE(readability-function-size) todo(iox-#832): make a struct out of arguments
+// @todo(iox-#832): make a struct out of arguments
+// NOLINTBEGIN(readability-function-size, bugprone-easily-swappable-parameters)
 UnixDomainSocket::UnixDomainSocket(const NoPathPrefix_t,
                                    const UdsName_t& name,
                                    const IpcChannelSide channelSide,
                                    const size_t maxMsgSize,
                                    const uint64_t maxMsgNumber IOX_MAYBE_UNUSED) noexcept
```

Comment:
Why not..?
```suggestion
                                   const uint64_t) noexcept
```

Reply:
Good point. I removed the parameter name here. The whole parameter will be probably removed with #832.